### PR TITLE
PP-515: Add script to change partner shares

### PR DESCRIPTION
**Pull request by antomor**
## What

- Add a utility script to be used to change the partner shares of a collector

## Why

- Without the script, we need to call directly the contract method, and it may be less user-friendly.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -73,6 +73,24 @@ E.g.:
...
+Pre-requirements: the collector we want to change must be deployed. See the [related section](#collector-deployment) to deploy a collector.
+
+The `change-partner-shares.js` is a utility script that can be run to change the partner shares of a collector already deployed. It receives the following parameters:
+- `collectorAddress`: mandatory, it's the address of the collector we want to change
+- `partnerConfig`: optional, it includes the new partner shares we want to set; if not specified we use `partner-shares.json` as default value; check the `partner-shares.sample.json` to verify the format.
+- `txGas`: optional, it's the gas limit used for the transaction. If the transaction fails, we may probably need to specify an higher value; default value is `150000`;
+
+Usage:
+```bash
+npx truffle --network <network_name> exec tasks/changePartnerShares.js --partnerConfig="<file_including_partners_config.json>" --collectorAddress="<collector_address>" --txGas="<gas_limit>"
```
Comment (franciscotobar) [inline Readme.md:87]:
```suggestion
npx truffle --network <network_name> exec tasks/change-partner-shares.js --partnerConfig="<file_including_partners_config.json>" --collectorAddress="<collector_address>" --txGas="<gas_limit>"
```

File: Readme.md [hunk 2]
```diff
@@ -73,6 +73,24 @@ E.g.:
...
+```
+Example:
+```bash
+npx truffle --network testnet exec tasks/changePartnerShares.js --partnerConfig="partner-shares.json" --collectorAddress="0x9b91c655AaE10E6cd0a941Aa90A6e7aa97FB02F4" --txGas="200000"
```
Comment (franciscotobar) [inline Readme.md:91]:
```suggestion
npx truffle --network testnet exec tasks/change-partner-shares.js --partnerConfig="partner-shares.json" --collectorAddress="0x9b91c655AaE10E6cd0a941Aa90A6e7aa97FB02F4" --txGas="200000"
```

**Comment by antomor** [comment]
@franciscotobar thank you for the comments, I've updated the PR.

> Quick question, updating the partner shares, does not obligate to withdraw the funds, right?

It's a pre-requirement, and indeed I specified it in the readme also, thank you for highlighting

**Review by ironFe93 — APPROVED** [review]
lgtm

**Review by franciscotobar — APPROVED** [review]
LGTM

**Pull request merged**